exercises/space-age/canonical-data.json

```diff
@@ -80,6 +80,16 @@
         "seconds": 1821023456
       },
       "expected": 0.35
+    },
+    {
+      "uuid": "57b96e2a-1178-40b7-b34d-f3c9c34e4bf4",
+      "description": "age on the Sun",
```

Comment:
Description could maybe indicate more the purpose of the test case, i.e. that only planets in our solar system are acceptable. 
The sun is only improtant because it reflects that case.
Maybe something like "Other heavenly bodies cause error" ?

Other than that this looks good to me. 👍